web/models/summary_helper.ex

```diff
@@ -0,0 +1,80 @@
+defmodule CoursePlanner.SummaryHelper do
+  @moduledoc """
+    Provides helper functions for the summary page
+  """
+  use CoursePlanner.Web, :model
+
+  alias CoursePlanner.{Repo, Terms.Term, OfferedCourse}
+
+  def get_term_offered_course_for_user(user_id, user_role) do
```

Comment:
it's gonna be one function for all users to get theiir personalized summary on term info